.vscode/settings.json

```diff
@@ -10,7 +10,7 @@
     "files.insertFinalNewline": true,
     "editor.formatOnSave": true,
     "editor.codeActionsOnSave": {
-        "source.fixAll": true
+        "source.fixAll": "explicit"

```

Comment:
VSCode keeps changing this automatically; `true` has been deprecated in favor of `"explicit"`